code/_expts/slotnick_etal_2001/analyze/plots/de_CreateSlotnickFigure1.m

```diff
@@ -4,25 +4,25 @@
 %  E1:  errorbar e
 %  taskTitle: slotnick task (captialized first letter)
 
-if ~exist('ax', 'var')
-    % Create figure
-    figure1 = de_NewFig(sprintf('%s_%s', stimSet, lower(taskTitle)));
-    ax = gca(); % make current figure the axis if not passed in.
-end
+    if ~exist('ax', 'var')
+        % Create figure
+        figure1 = de_NewFig(sprintf('%s_%s', stimSet, lower(taskTitle)));
+        ax = gca(); % make current figure the axis if not passed in.
+    end
 
-% Create errorbar
-errorbar(ax, Y1,E1,'MarkerSize', 15,...
-		 'MarkerFaceColor', [0.600000023841858 0.200000002980232 0],...
-		 'MarkerEdgeColor', [0.600000023841858 0.200000002980232 0],...
-		 'Marker', 'square',...
-		 'Color', [0 0 1]);
-box(ax, 'on');
-
-% Set the remaining axes properties
-set(ax, 'XTick', [1 2], 'XTickLabel', {'LH', 'RH'});
-ylabel(ax, 'Sum squared error');
-title(ax, sprintf('%s Stimuli: %s (n=%d)', taskTitle, stimSet, rons)); 
-
-
-figure1.handle = gcf();
+    % REVERSE the order of data, as Slotnick plots
+    % the *hemisphere injected*, whereas our data are for
+    % the hemisphere running.
+    errorbar(ax, Y1(end:-1:1), E1(end:-1:1),'MarkerSize', 15,...
+             'MarkerFaceColor', [0.600000023841858 0.200000002980232 0],...
+             'MarkerEdgeColor', [0.600000023841858 0.200000002980232 0],...
+             'Marker', 'square',...
+             'Color', [0 0 1]);
+    box(ax, 'on');
 
+    % Set the remaining axes properties
+    set(ax, 'XTick', [1 2], ...
+            'XTickLabel', {sprintf('LH'), sprintf('RH')});
+    xlabel('Hemisphere Injected');  % weird, right??
+    ylabel(ax, 'Sum squared error');
+    title(ax, sprintf('%s\n%s\n ', taskTitle, stimSet));
```

Comment:
The figure title was multi-line, and the length of the title made things ugly, so I made this multi-line. I also added a blank line at the bottom, as there was frequent overlap between the yaxis labeling and the title.